OMCompiler/Compiler/NSimCode/NSimCode.mo

```diff
@@ -115,7 +115,6 @@ public
       Integer jacobianIndex;
       Integer residualIndex;
       Integer implicitIndex; // this can be removed i think -> moved to solve
-      Integer genericCallIndex;
```

Comment:
Might be confusing that all indices are stored in this structure, just this index is missing.

Maybe remove all of these in the future? could possibly be tracked more easily.